src/Hangman/Domain/AggregateRoot/Game.aggregate.ts

```diff
@@ -54,7 +54,7 @@ export class Game extends AggregateRoot {
         ),
       );
     } catch (err) {
-      throw new MethodNotAllowedException('invalid game data');
+      throw new Error('invalid game data');
```

Comment:
deze message wordt nu geprint in de all-exceptions-filter, en in de json response geprint